lab_3_keywords_textrank/main.py

```diff
@@ -564,7 +669,7 @@ def get_top_keywords(self, n_keywords: int) -> tuple[int, ...]:
             tuple[int, ...]
                 top n most important tokens in the encoded text
         """
-        pass
+        return [key for (key, value) in sorted(self._scores.items(), key=lambda val: val[1], reverse=True)][:n_keywords]
```

Comment:
what if multiple tokens happen to have the same score?

Reply:
greater numbers should be placed first. I will try to handle this!

Reply:
are you sure? :eyes:

Reply:
i was wrong, lesser numbers should be placed first